pages/post/[slug].js

```diff
@@ -168,9 +175,7 @@ export default function Post() {
                   <div
                     ref={contentEl}
                     dangerouslySetInnerHTML={{
-                      __html: md({
-                        html: true,
-                      }).render(modifiedContent),
+                      __html: DOMPurify.sanitize(modifiedContent),
```

Comment:
Comments are not showing properly. Find a solution for it, Otherwise code is looking perfect